mephisto/operations/task_launcher.py

```diff
@@ -191,15 +191,24 @@ def _launch_limited_units(self, url: str) -> None:
                 if unit is None:
                     break
                 unit.launch(url)
+            if (
+                run_config is not None
+                and run_config.blueprint.link_task_source == False
+            ):
+                logger.info(
+                    "If you want your server to update on reload whenever you make changes to your webapp, then make sure to set \n\nlink_task_source: true\n\nin your task's hydra configuration and run \n\ncd webapp && npm run dev:watch\n\nin a separate terminal window. For more information check out:\nhttps://mephisto.ai/docs/guides/tutorials/custom_react/#12-launching-the-task"
+                )
```

Comment:
What is the rationale for putting the checking code here as opposed to somewhere else? Not challenging, just curious. Is there any other place this could be placed up the function call stack?

I find that adding a new param to the function just for this functionality to be a bit of a code smell, though I'm willing to be talked out of it.

Reply:
It seems to be messing up the python build test so I will probably have to find a new place to put it. 

I put it there so that I could get access to the run_config variables. I need to check if link_task is set to false before showing anything.

Reply:
e.g. why not put it in `launch_task_run_or_die`?

Reply:
Yeah, this seems like a way better solution. This will also prevent the tests from failing.